test/powershell/Language/Parser/Ast.Tests.ps1

```diff
@@ -5,20 +5,22 @@ Describe "The SafeGetValue method on AST returns safe values" -Tags "CI" {
         $HtAst = {
             @{ one = 1 }
             }.ast.Find({$args[0] -is $HashtableAstType}, $true)
+        $HtAst | Should Not BeNullOrEmpty
         $HtAst.SafeGetValue().GetType().Name | Should be Hashtable

```

Comment:
this extra test doesn't really get you that much. If $HtAst is null, or if GetType is null, or $HtAst.SafeGetValue() doesn't return a hashtable, the test will fail regardless.
Doesn't just `$HtAst.SafeGetValue() | should beoftype hashtable` suffice?